src/error-handling/error-contexts.md

```diff
@@ -5,25 +5,18 @@ contextual information to your errors and allows you to have fewer
 custom error types:
 
 ```rust,editable,compile_fail
-use std::{fs, io};
+use std::{fs::{self, File}, io};
```

Comment:
I'm still not sure why you change the imports like this :slightly_smiling_face:

The `self` seems unnecessary here — I normally follow the style of trying to minimize imports by importing a _module_, not individual items. That makes it much easier to see where things come from at a glance and it minimizes the necessary changes as you use more/less items from the module.